python/test/unit/language/test_matmul.py

```diff
@@ -118,6 +118,13 @@ def test_simple_matmul(dtype_src_str, dtype_dst_str, BLOCK_M, BLOCK_N, BLOCK_K,
         pytest.xfail("Skipping unsupported case")
     if "float32" in dtype_src_str and dtype_dst_str == "float16":
         pytest.xfail("Skipping unsupported case")
+    if dtype_dst_str == "bfloat16" and not is_xpu_cri():
+        pytest.xfail("Skipping unsupported case")
+    if is_xpu_cri():
+        if "float32" in dtype_src_str and dtype_dst_str == "bfloat16":
+            pytest.xfail("Skipping unsupported case")
+        if "float16" in dtype_src_str and dtype_dst_str == "bfloat16":
+            pytest.xfail("Skipping unsupported case")
```

Comment:
having a section for is_xpu is easier to merge from upstream
```suggestion
    if is_xpu():
       if dtype_dst_str == "bfloat16":
           if not is_xpu_cri():
              pytest.xfail("Skipping unsupported case")
           else:
              if dtype_src_str in ("float32", "float16"):
                 pytest.xfail("Skipping unsupported case")
```